default/scripting/techs/defense/Garrison.focs.txt

```diff
@@ -40,9 +40,9 @@ Tech
                 (LocalCandidate.LastTurnAttackedByShip < CurrentTurn)
             ]
             priority = [[AFTER_ALL_TARGET_MAX_METERS_PRIORITY]]
-            accountinglabel = "DEF_TECH_ACCOUNTING_LABEL"
```

Comment:
accountinglabel goes before priority: 
https://github.com/freeorion/freeorion/blob/master/default/scripting/buildings/IMPERIAL_PALACE.focs.txt#L123-L125
https://github.com/freeorion/freeorion/blob/master/parse/EffectParser.cpp#L121-L123

Reply:
yep, is what i need to read up about. i'd like maybe later to look into writing some test to make sure that everything is accounted for i.e. has an accounting label